__tests__/app.test.js

```diff
@@ -52,11 +52,32 @@ describe("GET /api/articles/:article_id", () => {
         expect(article).toHaveProperty("author");
         expect(article).toHaveProperty("title");
         expect(article).toHaveProperty("article_id");
-
+        expect(article).toHaveProperty("body")
         expect(article).toHaveProperty("topic");
         expect(article).toHaveProperty("created_at");
         expect(article).toHaveProperty("votes");
         expect(article).toHaveProperty("article_img_url");
       });
   });
 });
+describe('Error Handling 400/404', () => {
+  test("return 404 status, should return an error when the article is not found", () => {
+    return request(app)
+      .get("/api/articles/999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toHaveProperty("message");
+        expect(body.message).toBe("Article not found");
+      });
+  });
+  test("return 400 status, should return an error for non-existent article id", () => {
```

Comment:
The test description isn't quite right here. We're actually checking whether the article_id is _valid_ rather than whether it exists.